libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/utils/DirectionsResponseUtils.kt

```diff
@@ -3,16 +3,21 @@ package com.mapbox.navigation.base.internal.utils
 import com.mapbox.api.directions.v5.models.DirectionsResponse
 import com.mapbox.api.directions.v5.models.DirectionsRoute
 import com.mapbox.api.directions.v5.models.RouteOptions
-import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.CoroutineDispatcher
 import kotlinx.coroutines.withContext
 import org.json.JSONObject
 
+/**
+ * @throws Exception if we were unable to parse the route
+ */
+@Throws(Exception::class)
```

Comment:
I think we should be more specific here and below.

Reply:
Also it doesn't really help, because Kotlin doesn't force you to catch exceptions.